domains.txt

```diff
@@ -667,7 +746,9 @@ tda-digital.com
 tda-group.com
 tditrec.com
 team-prime.com
+teamconsulting-uk.com
 techid.co.uk
+technet-digital.co.uk
```

Comment:
. At TechNET Digital we aim to become the best boutique agency in the UK. This being done by providing both candidates & clients with the best experience within this digital era. From our corporate roots we are able to branch out to our vast network within both the UK & EU to make us the go to Digital agency to partner with. Having an array of sexy and up and coming brands and agencies in our portfolio it enhances us with our candidate outreach.